testnet/install.rst

```diff
@@ -63,9 +63,9 @@ Otherwise, if you wish to run a Massa node from source code, here are the steps
 On Ubuntu / MacOS
 ~~~~~~~~~~~~~~~~~
 
-- on Ubuntu, these tools and libs must be installed: ``sudo apt install pkg-config curl git build-essential libssl-dev
-  libclang-dev protobuf-compiler``
-- on MacOS: ``brew install llvm protobuf``
+- on Ubuntu, these libs must be installed: ``sudo apt install pkg-config curl git build-essential libssl-dev cmake 
+  libclang-dev cmake protobuf-compiler``
+- on MacOS: ``brew install llvm cmake protobuf``
```

Comment:
There is different variants ... redirect to https://github.com/massalabs/massa/blob/main/massa-grpc/README.md